i18n/sv.toml

```
@@ -0,0 +1,85 @@
+[language]
+other = "sv"
+
+[links]
+  [links.get-involved]
+  other = "Bli involverad"
+
+  [links.get-in-touch]
+  other = "Kom i kontakt"
+
+  [links.go-home]
+  other = "Gå hem"
+
+  [links.close-menu]
+  other = "Stäng meny"
+
+  [links.documentation]
+  other = "Dokumentation"
+
+  [links.join-the-discussion]
+  other = "Gå med i diskussionen"
+
+[socials]
+  [socials.twitter]
+  other = "Twitter"
+
+  [socials.reddit]
+  other = "Reddit"
+
+  [socials.forums]
+  other = "Forum"
+
+  [socials.slack]
+  other = "Slack"
+
+  [socials.github]
+  other = "Github"
+
+[copyright]
+  [copyright.line1]
+  other = "© 2020 The Rocky Linux Projektet. Alla rättigheter förbehållna."
```

```suggestion
  other = "© 2020 Rocky Linux-projektet. Alla rättigheter förbehållna."
```
Or maybe just keep it as "The Rocky Linux Project" since that's it's name, other translations have handled it differently.